apps/yapms/src/routes/+page.server.ts

```diff
@@ -1 +1,30 @@
+import type { PageServerLoad } from './$types';
+import { globSync } from 'glob';
+import fs from 'fs';
+
+export const load: PageServerLoad = () => {
+	const files = globSync('./src/lib/assets/maps/**/*.svg');
+
+	const search = [];
+
+	for (const file of files) {
+		const data = fs.readFileSync(file, 'utf8');
+		const title = data.match(/title=['"](?<title>[^'"]*)['"]/)?.groups?.['title'];
+		if (title === undefined) {
+			continue;
+		}
+		const route = '/app/' + file.split('/').pop()?.split('.').at(0)?.replaceAll('-', '/');
+		search.push({
```

Comment:
the file list is "backwards" so i find that replacing this search.push with a search.unshift gives us far better search results (e.g when you type in "2024 pres" it takes you to the normal 2024 presidential map instead of the take-all map

Reply:
Nice catch!